src/data_management/rasters/RasterForm.js

```diff
@@ -659,539 +118,257 @@ class RasterFormModel extends Component {
+        />
+        <SelectBox
+          name="accesModifier"
+          title="Authorization type"
+          subtitle="Specify who can view this raster"
+          choices={[
+            [
+              "Private",
+              "Private",
+              "Just the selected organisations can view"
+            ],
+            [
+              "Common",
+              "Common",
+              "All logged in users can view"
+            ],
+            [
+              "Public",
+              "Public",
+              "All users can view"
+            ] 
+          ]}
+          validators={[required("Please select an acces modifier.")]}
+          initial ={
+            (
+              currentRaster && 
+              currentRaster.access_modifier
+            ) ||
+            "Private" 
+          }
+          showSearchField={false}
+        />
+        <TextInput
+          name="rasterName"
+          title="Name of this Raster"
+          placeholder="Name of this Raster"
+          validators={[minLength(5)]}
+          initial = {
+            currentRaster && 
+              currentRaster.name
+          }
+        />
+        <TextArea
+          name="description"
+          placeholder="Fill in your description here"
+          subtitle="Add a clear description of this raster with reference to the data source."
+          validators={[minLength(1)]}
+          initial = {
+            currentRaster && 
+              currentRaster.description
+          }
+        />
+        <SelectBox
+          name="aggregationType"
+          title="Aggregation Type"
+          subtitle="Specify how data should be displayed in region selection mode"
+          placeholder="click to select aggregation type"
+          choices={[
+            [
+              "none",
+              "none",
+              "no aggregation"
+            ],
+            [
+              "counts",
+              "counts",
+              "area per category"
+            ],
+            [
+              "curve",
+              "curve",
+              "cumulative distribution"
+            ],
+            [
+              "sum",
+              "sum",
+              "values in the region are summed"
+            ],
+            [
+              "average",
+              "average",
+              "values in the region are averaged"
+            ]
+          ]}
+          validators={[required("Please select an aggregation type.")]}
+          showSearchField={false}
+          initial = {
+            (
+            currentRaster && 
+            currentRaster.aggregation_type
+            ) || null
+          }
+        />
+        <SelectBox
+          name="observationType"
+          title="Observation Type"
+          subtitle="Specify the physical quantity and unit of the data"
+          placeholder="click to select observation type"
+          choices={this.props.observationTypes.available.map(obsT=>
+            [obsT.code, obsT.code]
```

Comment:
Display moet een combinatie van parameter, unit en code zijn, wsl.

Reply:
In de huidige schermen is dit code.
Heb Joeri de vraag gesteld in JIRA.

Reply:
Volgens Lex was het idd goed. Done